src/ClientData/include/ClientData/TimerDataInterface.h

```diff
@@ -19,6 +19,12 @@ struct TimerMetadata {
   uint32_t process_id;
 };
 
+// Free Function that will be used for any implementation of
+// TimerDataInterface::GetTimersAtDepthDiscretized to get the next pixel timestamp.
+[[nodiscard]] uint64_t GetNextPixelBoundaryTimeNs(uint64_t current_timestamp_ns,
```

Comment:
Since this is now in a header, do you think this can be tested with some unit tests? :innocent: 

(Not saying that it needs to be done with this PR, but if it was just little work for you it would be very much appreciated. :-))

Reply:
I'm testing the function that use this method, but you are right, it's better to test both :)

http://b/243628546